internal/backend/db/dbgorm/health.go

```diff
@@ -0,0 +1,7 @@
+package dbgorm
+
+func (db *gormdb) Report() []error {
+	s, _ := db.db.DB()
+	err := s.Ping()
```

Comment:
Ping is not enough since it doesn't check permissions. Add a `SELECT 1` query.

Reply:
select 1 doesn't check permissions as well, it only check I can select from a specific schema and specific table